apps/web/components/apps/AppCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { StarIcon } from "@heroicons/react/solid";
+import Link from "next/link";
+
+import Button from "@components/ui/Button";
+
+interface AppCardProps {
+  logo: string;
+  name: string;
+  slug?: string;
+  category?: string;
+  description: string;
+  rating: number;
+  reviews?: number;
+}
+
+export default function AppCard(props: AppCardProps) {
+  return (
+    <Link href={"/apps/" + props.slug}>
+      <a className="block rounded-sm border border-gray-300 p-5 hover:bg-neutral-50">
+        <div className="flex">
+          <img src={props.logo} alt={props.name + " Logo"} className="mb-4 h-12 w-12 rounded-sm" />
+          <Button
+            color="secondary"
+            className="ml-auto flex self-start"
+            onClick={() => {
+              // TODO: Actually add the integration
+              console.log("The magic is supposed to happen here");
```

Comment:
If we go to installed, when will the specific logic for each app be triggered? I would think the user needs to go to the config app logic or whatever it might be for each app when hitting add. cc @PeerRich @baileypumfleet

Reply:
In terms of the flow, it makes sense to make the "magic" happen here as otherwise if the user has to go to the specific app page for installation, the add button doesn't really account for much. Might as well rename it to something else, or just remove it and make the entire card clickable which takes the user to the app-specific installation page